util/envDynawo.sh

```diff
@@ -324,30 +324,38 @@ set_environment() {
   export_var_env_force DYNAWO_THIRD_PARTY_INSTALL_DIR_VERSION=$DYNAWO_THIRD_PARTY_INSTALL_DIR/$DYNAWO_BUILD_TYPE_THIRD_PARTY$SUFFIX_CX11
 
   if [ -d "$DYNAWO_THIRD_PARTY_INSTALL_DIR_VERSION/boost" ]; then
-    export_var_env DYNAWO_BOOST_HOME=$DYNAWO_THIRD_PARTY_INSTALL_DIR_VERSION/boost
-    unset DYNAWO_BOOST_HOME_DEFAULT
-    if [ "$DYNAWO_LIBRARY_TYPE" = "STATIC" ]; then
-      if [ "$DYNAWO_BOOST_HOME" = "$DYNAWO_THIRD_PARTY_INSTALL_DIR_VERSION/boost" ]; then
-        export_var_env_force DYNAWO_BOOST_USE_STATIC=ON
+    if [ ! -z "$(ls -A $DYNAWO_THIRD_PARTY_INSTALL_DIR_VERSION/boost)" ]; then
+      export_var_env DYNAWO_BOOST_HOME=$DYNAWO_THIRD_PARTY_INSTALL_DIR_VERSION/boost
+      unset DYNAWO_BOOST_HOME_DEFAULT
+      if [ "$DYNAWO_LIBRARY_TYPE" = "STATIC" ]; then
+        if [ "$DYNAWO_BOOST_HOME" = "$DYNAWO_THIRD_PARTY_INSTALL_DIR_VERSION/boost" ]; then
+          export_var_env_force DYNAWO_BOOST_USE_STATIC=ON
+        fi
       fi
     fi
   fi
 
   if [ -d "$DYNAWO_THIRD_PARTY_INSTALL_DIR_VERSION/libarchive" ]; then
-    export_var_env DYNAWO_LIBARCHIVE_HOME=$DYNAWO_THIRD_PARTY_INSTALL_DIR_VERSION/libarchive
-    unset DYNAWO_LIBARCHIVE_HOME_DEFAULT
+    if [ ! -z "$(ls -A $DYNAWO_THIRD_PARTY_INSTALL_DIR_VERSION/libarchive)" ]; then
```

Comment:
Why not using the -d similarly to above?

Reply:
it checks if folder exists and is not empty (something has been installed)